openverse_catalog/dags/common/dag_factory.py

```diff
@@ -71,8 +126,15 @@ def create_provider_api_workflow(
                       be run (if `dated=True`).
     dagrun_timeout:   datetime.timedelta giving the total amount of time
                       a given dagrun may take.
+    doc_md:           string which should be used for the DAG's documentation markdown
+    media_types:      list describing the media type(s) that this provider handles
+                      (e.g. ("image",), ["audio"], ["image", "audio"], etc.)
```

Comment:
Mixing the types in the documentation is confusing to me for whatever reason... or at least it took a little longer of reading it to understand what was going on. Maybe some back ticks would help?

```suggestion
                      (e.g. `("image",)`, `["audio"]`, `["image", "audio"]`, etc.)
```

It already being nested inside of grammatical parenthesis makes the tuple version not stand out as much.

Reply:
Ah, totally fair. My intent here was to point out that any sequence type is accepted. We're using a tuple as the default argument because lists are mutable and mutable default arguments are probably the most common "gotcha" in Python[^1]. If you think that this example is better served by consistent types, I'm happy to change it!

[^1]: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
I do think consistent types would be good as well as the back-ticks to help delineate the code parens from the grammatical ones. If anything, always using a tuple at least communicates to the reader that the list will not be modified.